eng/pipelines/branch-cleanup.yml

```diff
@@ -22,13 +28,37 @@ jobs:
       - ${{ each repo in parameters.Repos }}:
         - task: PowerShell@2
           displayName: Clean Up Sync Common Branches
-          condition: succeeded()
+          condition: succeededOrFailed()
+          continueOnError: true
+          inputs:
+            pwsh: true
+            workingDirectory: $(System.DefaultWorkingDirectory)
+            filePath: $(System.DefaultWorkingDirectory)/eng/common/scripts/Delete-RemoteBranches.ps1
+            arguments: >
+              -RepoId ${{ repo }}
+              -BranchPrefix "sync-eng/common-" 
+              -AuthToken $(azuresdk-github-pat)
+           
+      - pwsh: | 
+          $expirationDate = (Get-Date).AddDays(-7)
```

Comment:
I assume this was for testing and should be removed now.

Reply:
After a second thought, it is weird to add extra step for testing use. I removed the step.